ONSdigital/rm-action-service
# Action simplification and final action exporter integration

# Motivation and Context


# What has changed



Simplifying the action service and removing the action exporter code. 

Action will now generate both the emails and the print files when the distribute action cron job runs, if there are any that need processing. This jobs runs every 5 minutes.

Also reducing the action execution cron job to run every 15 minutes to match the values set by the UI. This should hopefully reduce the CPU usage on action service.

# How to test?
Create a collection exercise and ensure all events, i.e. mps, go live, reminders generate the correct actions i.e. create print files or send emails.

# Links
The design for this can be view here: https://github.com/ONSdigital/ras-rm-documentation/tree/master/service-design/improvements/action-service

https://trello.com/c/UiseNGbO/195-s23-action-service-simplify-action-exporter-functionality
